src/draw/shapes.typ

```diff
@@ -921,6 +921,8 @@
         aabb-height,
         border.segments,
         typst-rotate(angle,
+          reflow: true,
+          origin: center + horizon,
```

Comment:
`center + horizon` is the default, but we should force override it here to prevent show-rules changing cetz' `content` behaviour.